docs/docs/using-pants/key-concepts/backends.mdx

```diff
@@ -98,7 +98,8 @@ The list of all backends (both stable and experimental) is also available via `p
 | `pants.backend.experimental.python.framework.django`               | Enables better support for projects using Django: [https://www.djangoproject.com](https://www.djangoproject.com)                                      |                                                                                                       |
 | `pants.backend.experimental.python.framework.stevedore`            | Enables better support for projects using stevedore: [https://docs.openstack.org/stevedore/](https://docs.openstack.org/stevedore/)                   |                                                                                                       |
 | `pants.backend.experimental.python.lint.add_trailing_comma`        | Enables add-trailing-comma, a Python code formatter: [https://github.com/asottile/add-trailing-comma](https://github.com/asottile/add-trailing-comma) | [`add-trailing-comma`](../../../reference/subsystems/add-trailing-comma.mdx)                          |
-| `pants.backend.experimental.python.lint.ruff`                      | Enables Ruff, an extremely fast Python linter: [https://beta.ruff.rs/docs/](https://beta.ruff.rs/docs/)                                               | [Linters and formatters](../../python/overview/linters-and-formatters.mdx)                            |
+| `pants.backend.experimental.python.lint.ruff`                      | Enables Ruff, an extremely fast Python linter: [https://docs.astral.sh/ruff/linter/](https://docs.astral.sh/ruff/linter/)                             | [Linters and formatters](../../python/overview/linters-and-formatters.mdx)                            |
```

Comment:
Perhaps add a note that it will enable linting only.. for those that know it has features for more than that.
```suggestion
| `pants.backend.experimental.python.lint.ruff`                      | Enables Ruff (for `lint`), an extremely fast Python linter: [https://docs.astral.sh/ruff/linter/](https://docs.astral.sh/ruff/linter/)                             | [Linters and formatters](../../python/overview/linters-and-formatters.mdx)                            |
```